docs/index.rst

```diff
@@ -12,10 +15,17 @@ designed to facilitate packaging Python projects.
    build_meta
    pkg_resources
    references/keywords
-   roadmap
    setuptools
+
+.. toctree::
```

Comment:
This is just a suggestion on how to organise the sidebar in 2 logical sections: one that targets directly the end user and one that is more "meta" regarding the project...